subsys/event_manager/event_manager.c

```diff
@@ -59,11 +61,20 @@ static void event_processor_fn(struct k_work *work)
 
 		trace_event_execution(eh, true);
 		if (IS_ENABLED(CONFIG_DESKTOP_EVENT_MANAGER_SHOW_EVENTS)) {
```

Comment:
make sure `DESKTOP_EVENT_MANAGER_SHOW_EVENTS` depends on LOG

Reply:
done (same for `DESKTOP_EVENT_MANAGER_SHOW_LISTENERS`)

In addition added `depends on DESKTOP_EVENT_MANAGER_SHOW_EVENTS`  for `DESKTOP_EVENT_MANAGER_SHOW_EVENT_HANDLERS` - without `DESKTOP_EVENT_MANAGER_SHOW_EVENTS`  it has no effect